scripts/plugins/openvswitch-config-update

```diff
@@ -75,8 +72,7 @@ def update(session, args):
 
     if not controller and currentControllers:
         delete_cacert()
-        cmd = [iptablectl, "close", "6640"]
-        output = subprocess.Popen(cmd, stdout=subprocess.PIPE).communicate()
```

Comment:
I believe the communicate is there to wait until the execution of iptablectl finishes, have you checked that runCmd waits as well?

Reply:
The `communicate` is also used in runCmd, refer to https://github.com/xenserver/python-libs/blob/master/xcp/cmd.py#L70.